### PR TITLE
Add descriptions for entries

### PR DESCRIPTION
**Pull request by platy11**
Adds descriptions for 11 entries in the list. Helps with #928.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -202,7 +202,7 @@
 - [Material Design](https://github.com/sachin1092/awesome-material)
 - [D3](https://github.com/wbkd/awesome-d3) - Library for producing dynamic, interactive data visualizations.
 - [Emails](https://github.com/jonathandion/awesome-emails)
-- [jQuery](https://github.com/petk/awesome-jquery)
+- [jQuery](https://github.com/petk/awesome-jquery) - JavaScript library designed for ease of use.
```
Comment (sindresorhus) [inline readme.md]: This description is too generic. It should describe what jQuery is.

File: readme.md [hunk 2]
```diff
@@ -373,7 +373,7 @@
 - [Science Fiction](https://github.com/sindresorhus/awesome-scifi) - Scifi.
 - [Fantasy](https://github.com/RichardLitt/awesome-fantasy)
 - [Podcasts](https://github.com/guipdutra/awesome-geek-podcasts)
-- [Email Newsletters](https://github.com/vredniy/awesome-newsletters)
+- [Email Newsletters](https://github.com/vredniy/awesome-newsletters)
```
Comment (sindresorhus) [inline readme.md]: Don't add whitespace.

File: readme.md [hunk 3]
```diff
@@ -487,7 +487,7 @@
 
 ## Miscellaneous
 
-- [JSON](https://github.com/burningtree/awesome-json)
+- [JSON](https://github.com/burningtree/awesome-json) - Object notation system.
```
Comment (sindresorhus) [inline readme.md]: This description needs to be more specific. It doesn't really explain anything.

File: readme.md (hunk 2 above)
Comment (platy11) [reply readme.md]: Sorry, that was a mistake.

**Comment by sindresorhus** [comment]
Looks good now. Thanks for your contribution :)

**Pull request merged**